controls/radvirtualgrid/mvvm.md

```diff
@@ -0,0 +1,327 @@
+
+## Defining the business model
+
+Firstly, we need an object model that will be used for further populating __RadVirtualGrid__ with data. For this purpose, the following __Club__ object is defined.
+
+#### __[C#] Example 1: Defining the Club object__
+
+{{region radvirtualgrid-mvvm_0}}
+	public class Club : INotifyPropertyChanged
+    {
+        public event PropertyChangedEventHandler PropertyChanged;
+
+        private string name;
+        private DateTime established;
+        private int stadiumCapacity;
+        private int primaryKey;
+
+        public int PrimaryKey
+        {
+            get { return this.primaryKey; }
+            set
+            {
+                this.primaryKey = value;
+                this.OnPropertyChanged("PrimaryKey");
+            }
+        }
+        public string Name
+        {
+            get { return this.name; }
+            set
+            {
+                if (value != this.name)
+                {
+                    this.name = value;
+                    this.OnPropertyChanged("Name");
+                }
+            }
+        }
+
+        public DateTime Established
+        {
+            get { return this.established; }
+            set
+            {
+                if (value != this.established)
+                {
+                    this.established = value;
+                    this.OnPropertyChanged("Established");
+                }
+            }
+        }
+
+        public int StadiumCapacity
+        {
+            get { return this.stadiumCapacity; }
+            set
+            {
+                if (value != this.stadiumCapacity)
+                {
+                    this.stadiumCapacity = value;
+                    this.OnPropertyChanged("StadiumCapacity");
+                }
+            }
+        }
+
+        public Club()
+        {
+
+        }
+
+        public Club(string name, DateTime established, int stadiumCapacity)
+        {
+            this.name = name;
+            this.established = established;
+            this.stadiumCapacity = stadiumCapacity;
+        }
+
+        protected virtual void OnPropertyChanged(PropertyChangedEventArgs args)
+        {
+            PropertyChangedEventHandler handler = this.PropertyChanged;
+            if (handler != null)
+            {
+                handler(this, args);
+            }
+        }
+
+        private void OnPropertyChanged(string propertyName)
+        {
+            this.OnPropertyChanged(new PropertyChangedEventArgs(propertyName));
+        }
+
+        public override string ToString()
+        {
+            return this.Name;
+        }
+
+        public static ObservableCollection<Club> GetClubs()
+        {
+            ObservableCollection<Club> clubs = new ObservableCollection<Club>();
+            Club club;
+            int id = 0;
```

Comment:
Why we use the 'id' variable can't be just hardcode the numbers? Also, do we need the comments with the club names above each variable. Can we remove those? 

In general we want the code in our SDK and especially in the documentation to be as tight as it can be. Shorter snippets, better. Of course this is valid as long as we don't decrease the quality of the provided information with it.